README.md

```diff
@@ -21,6 +21,7 @@
 			- [`--details`](#--details)
 			- [`--format`](#--format)
 			- [`--fix`](#--fix)
+				- [Dry Run Mode](#dry-run-mode)
```

Comment:
@marianfoo Dry run is not part of this PR, isn't?

Reply:
yeah sorry, a extension in VS Code always added that. I deleted it.